pubspec.yaml

```diff
@@ -1,6 +1,7 @@
 name: directus_api_manager
 description: Communicate with a Directus server using its REST API.
 version: 1.7.2
```

Comment:
You should increase the version number to 1.7.3 and also update the changelog.md to document this new feature

Reply:
Increasing to 1.7.3 the version and added to the changelog.md file. We are missing 1.7.2 changes in the changelog. Should it not be version 1.8.0 as a 'new feature with no breaking changes' (even if this is a small new feature)?

Reply:
Good catch, 1.8.0 is better in this case.
My bad on 1.7.2 changelog missing, I'll add it after your commit to spare you a git conflict

Reply:
I have tagged it v1.7.3 now, but next time I will remember the convention.